charts/kubernikus-system/charts/prometheus/templates/_prometheus.yaml.tpl

```diff
@@ -241,7 +241,36 @@ scrape_configs:
       target_label: __metrics_path__
       replacement: /api/v1/nodes/${1}:4194/proxy/metrics
 
-# Static Targets
+- job_name: 'blackbox-ingress'
+  metrics_path: /probe
+  params:
+    module: [http_2xx]  # Look for a HTTP 200 response.
```

Comment:
only `2xx` responses are ok for now

Reply:
I think this will probably not work for these https://github.com/sapcc/kubernikus/blob/master/charts/kube-master/templates/ingress.yaml

We should maybe also have a module that additionally accepts 40[13]. Can we maybe have this overridable by some annotation on the individual ingress spec? So that `http_2xx` is the default but  you can override that by specifying the module in an annotation.